### DISCUSSION TITLE
Model View WEBXR or SCENE VIEWER in WebView (Flutter)

### DISCUSSION THREAD

**Discussion by sandeeep10thakur** [google/model-viewer#4250] [Q&A]
### Description

I am not able to use Model Viewer's WebXR - AR mode in Webview. It takes the screen to CORE AR update screen (Although my CoreAR is already updated to latest Version). Then the AR button also disappers.

I tried SceneViewer - AR in one of my Nokia device (Nokia 6.1 Plus) inside Webview and it worked. 
But when I tried SceneViewer AR on other mobile devices it doesn't work.

### Browser Affected
Webview (Flutter/Kodular/App Inventor)

### AR

- [ ] WebXR
- [ ] SceneViewer

**Comment by elalish** [comment]
Indeed, WebXR is not yet supported in WebViews. Please file a request with Chromium.

**Comment by vulamavi** [comment]
Model Viewer use webXR. It active AR in webview.
Or you can hire me to resolve your problem. :))
